syntax = "proto2";

package jungi.mobgtw.proto;
option optimize_for = LITE_RUNTIME;

message DevicesListRequest {}
